missing installation instructions (Linux)

**Describe the bug**
The crate depends on the external ncurses, openssl and sqlite3 libraries. These dependencies have to be installed manually because they are C libraries and are thus not installed by cargo.

**To Reproduce**
1. run `cargo install ncgopher` as per the current installation instructions
2. see build error about ncurses
3. install ncurses
4. rerun `cargo install ncgopher`
5. see build error about openssl
6. install openssl
7. rerun `cargo install ncgopher`
8. see linker error "cannot find -lsqlite3"
9. install sqlite3
10. rerun `cargo install ncgopher`
11. finally installed

**Expected behavior:**
Install in one go without tracking down which dependencies to install.

**Desktop:**
 - OS: Linux Debian
 - Version 4.19146-1

**Additional context**
When using apt, the libraries can be installed with `sudo apt install pkg-config libncurses-dev libssl-dev libsqlite3-dev`.

--- a/README.md
+++ b/README.md
@@ -47,8 +47,26 @@ Obligatory screenshot:
 
 ## Installation
 
-`ncgopher` has no fancy installation process right now. Make sure to install
-rust on your system, but the latest release can be installed through cargo:
+`ncgopher` has no fancy installation process right now. There are some external
+dependencies which have to be installed. First and foremost you will of course
+need to have Rust installed. Further dependencies are the *openssl*, *ncurses*
+and *sqlite3* libraries. If these are not installed, the build will fail but
+you will most likely be able to tell what is missing.
+
+---
+### Debian-based Linux
+
+    sudo apt install build-essential pkg-config libssl-dev libncurses-dev libsql3-dev
+
+### Arch-based Linux
+
+    sudo pacman -S base-devel pkg-config openssl ncurses sqlite
+---
+
+If you know how to install the listed dependencies on your operating system and it is
+not listed, please make a pull request to add it.
+
+After installing these dependencies run
 
     cargo install ncgopher
 

